packages/tools/pie-wrapper-react/scripts/react-remove-wrapper.js

```diff
@@ -0,0 +1,33 @@
+import { unlink, existsSync } from 'fs';
+
+// removes react wrapper from index.ts after dist has built
+export default function removeReactWrapper (customElementsObject) {
+    let components = []
+    let removedWrapper = ''
```

Comment:
`removedWrapper` hasn't been reassigned after it was declared? is this intended?